src/devices/halemeier.ts

```diff
@@ -56,6 +56,16 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['HA-ZX1'],
+        model: 'HA-ZX1',
+        vendor: 'Halemeier GmbH',
+        description: 'X-Mitter Smart Remote Control',
+        extend: [batteryPercentage(), identify()],
+        fromZigbee: [fz.command_off, fz.command_on, fz.command_stop, fz.command_move],
+        exposes: [e.battery().withAccess(ea.STATE_GET), e.action(['recall_*', 'on', 'off', 'brightness_move_up', 'brightness_move_down']),
```

Comment:
`e.battery()` shouldn't be needed here (already covered by `batteryPercentage()`)

Reply:
you are right Koenkk